scripts/tests/sanitycheck/test_sanitycheck.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+This test file contains foundational testcases for Sanitycheck tool
+"""
+
+import os
+import imp
+import sys
+import pytest
+ZEPHYR_BASE = os.getenv("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/"))
+from sanity_chk import scl
+sanitycheck = imp.load_source('sanitycheck', ZEPHYR_BASE + '/scripts/sanitycheck')
+
+@pytest.fixture(name='test_data')
+def _test_data():
+    """ Pytest fixture to set the path of test_data"""
+    data = ZEPHYR_BASE + "/scripts/tests/sanitycheck/test_data/"
+    return data
+
+def test_zephyr_base_set():
+    """ Test to check if ZEPHYR_BASE environment variable is set """
+    assert os.getenv("ZEPHYR_BASE") is not None
+
+def test_yamlload():
+    """ Test to check if loading the non-existent files raises the errors """
+    filename = 'testcase_nc.yaml'
+    with pytest.raises(FileNotFoundError):
+        scl.yaml_load(filename)
+
+def test_correct_testcase_schema(test_data):
+    """ Test to validate the testcase schema"""
+    filename = test_data + 'testcase_correct.yaml'
```

Comment:
Maybe good for this initial PR, but the file names need to indicate what they are testing, for example here, you are testing the schema, so testcase_schema.yaml...

Reply:
Is renaming the file to "testcase_correct_schema.yaml" fine? Since I plan to add an additional testcase which tests if the error is raised if the yaml file is incorrect/not in the desired format and would use "testcase_incorrect_schema.yaml" filename for that.